test/jest.setup.js

```diff
@@ -0,0 +1,9 @@
+import fetchMock from 'fetch-mock';
+import { Headers, Request, Response } from 'cross-fetch';
+
+fetchMock.setImplementations({
```

Comment:
Here we explicitly make sure that `fetchMock` is using expected version of `node-fetch` and not other `node-fetch` present in our dependency treee.